Flutter_TDD_Architecture_Course_notes.md

```diff
@@ -1 +1,271 @@
 
+## Notes for Flutter TDD Clean Architecture Course Videos
+
+#### 1. Explanation & Project Structure
+> Video:https://www.youtube.com/watch?v=KjE2IDphA_U&list=PLB6lc7nQ1n4iYGE_khpXRdJkJEp9WOech
```

Comment:
Space be video and link so that it is clickable:
```suggestion
> Video playlist: https://www.youtube.com/watch?v=KjE2IDphA_U&list=PLB6lc7nQ1n4iYGE_khpXRdJkJEp9WOech
```